packages/roosterjs-content-model/lib/domToModel/processors/containerProcessor.ts

```diff
@@ -0,0 +1,28 @@
+import { ContentModelBlockGroup } from '../../publicTypes/block/group/ContentModelBlockGroup';
+import { generalBlockProcessor } from './generalBlockProcessor';
+import { generalSegmentProcessor } from './generalSegmentProcessor';
+import { isBlockElement } from 'roosterjs-editor-dom';
+import { isNodeOfType } from '../../domUtils/isNodeOfType';
+import { NodeType } from 'roosterjs-editor-types';
+import { textProcessor } from './textProcessor';
+
+/**
+ * @internal
+ */
+export function containerProcessor(group: ContentModelBlockGroup, parent: ParentNode) {
```

Comment:
Do we use the ElementProcessor Type in this function and the other function Element processors or it is not needed?

Reply:
Yes, it should be. Let me solve it later.